lib/Twig/Extension/Core.php

```diff
@@ -817,9 +817,10 @@ function twig_join_filter($value, $glue = '')
  *  {# returns [aa, bb, cc] #}
  * </pre>
  *
- * @param string $value     A string
- * @param string $delimiter The delimiter
- * @param int    $limit     The limit
+ * @param Twig_Environment $env       A Twig_Environment instance
+ * @param string           $value     A string
+ * @param string           $delimiter The delimiter
+ * @param int              $limit     The limit
  *
```

Comment:
I like the minimal docs approach, but I think all params should be in the docs or non, since missing params confuses IDE's/SCA (and by that developers ;) )